R/coord-sf.R

```diff
@@ -255,19 +255,24 @@ CoordSf <- ggproto("CoordSf", CoordCartesian,
     graticule$y_start <- sf_rescale01_x(graticule$y_start, y_range)
     graticule$y_end <- sf_rescale01_x(graticule$y_end, y_range)
 
-    list(
+    list2(
       x_range = x_range,
       y_range = y_range,
       graticule = graticule,
       crs = params$crs,
       default_crs = params$default_crs,
-      viewscales = viewscales
+      !!viewscales
```

Comment:
This makes `CoordSf$setup_panel_guides()` redundant and can rely on parental `Coord$setup_panel_guides()`.